db/virtual_table.hh

```diff
@@ -53,7 +53,7 @@ public:
 class memtable_filling_virtual_table : public virtual_table {
     dirty_memory_manager _dmm;
 public:
-    using virtual_table::virtual_table;
+    explicit memtable_filling_virtual_table(schema_ptr s, logalloc::tracker& tracker) : virtual_table(std::move(s)), _dmm(tracker) { }
```

Comment:
Similarly here. Pass a memtable-creating function instead of logalloc::tracker which is too low-level here.